api/test/import-data/xlsx-uploads/import-mocks.ts

```diff
@@ -8,19 +8,19 @@ async function createIndicatorsForXLSXImport(): Promise<string[]> {
   const indicatorSpec = [
     {
       name: 'Deforestation loss due to land use change',
-      nameCode: 'DF_LUC_T',
+      nameCode: 'deforestation',
     },
     {
       name: 'Carbon emissions due to land use change',
-      nameCode: 'GHG_LUC_T',
+      nameCode: 'carbon_emissions',
     },
     {
       name: 'Biodiversity loss due to land use change',
-      nameCode: 'BL_LUC_T',
+      nameCode: 'biodiversity_loss',
     },
     {
       name: 'Unsustainable water use',
-      nameCode: 'UWU_T',
+      nameCode: 'unsustainable_water_use',
     },
   ];
```

Comment:
You can also use the LAYER_TYPE enum for the tests

Reply:
Hm weird, the issue does not appear as outdated even tho I see you made the suggested change 🤔